ibm/service/power/data_source_ibm_pi_dhcps.go

```diff
@@ -48,7 +48,8 @@ func DataSourceIBMPIDhcps() *schema.Resource {
 						Attr_DhcpNetworkDeprecated: {
 							Type:        schema.TypeString,
 							Computed:    true,
-							Description: "The ID of the DHCP Server private network (deprecated - replaced by network_id)",
+							Description: "The ID of the DHCP Server private network",
+							Deprecated:  "This field is deprecated, use network_id instead",
```

Comment:
If anything we might have to remove this altogether since it has been deprecated for a while. I am not sure if we return it or not to the user.

Reply:
Removed deprecated fields.